src/universe/UniverseDeclareAnnotationMirror.java

```diff
@@ -0,0 +1,36 @@
+package universe;
+
+import org.checkerframework.framework.source.SourceChecker;
+import universe.qual.Any;
+import universe.qual.Bottom;
+import universe.qual.Lost;
+import universe.qual.Peer;
+import universe.qual.Rep;
+import universe.qual.Self;
+import org.checkerframework.dataflow.qual.Pure;
+import org.checkerframework.javacutil.AnnotationBuilder;
+
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.util.Elements;
+
+/**
+ * Class that declares the AnnotationMirrors used by typecheck and inference
+ *
+ */
+
+public class UniverseDeclareAnnotationMirror {
```

Comment:
How about `UniverseAnnotationMirrorHolder`? The `Declare` sounds odd.
I'm still not quite sold on this class, in particular as we can't make the fields final. Using static fields is also not really nice.
Wouldn't it be just as simple to have these fields (even final!) in the respective checkers? There will be a bit of duplication, but it will be clearer and less error-prone.